### PR TITLE
Bookstore

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
# Bookstore

In this project I:
- configure Redux store
- Import CreateStore, combineReducers, and applyMiddleWare
- Install redux-logger
- Implement Add and remove book

### CODE REVIEW

File: src/components/book/AddBook.js [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+  return (
+    <FormWrapper id="add-books">
+      <Headings>ADD NEW BOOK</Headings>
+      <FormContainer onSubmit={submitBookToStore}>
+        <AddBookInput placeholder="Book title" name="title" />
+        <AddBookInput placeholder="Book author" name="author" />
+        <CategoryInput placeholder="Not available" />
+        <Button type="submit">ADD BOOK</Button>
+      </FormContainer>
+    </FormWrapper>
```
Comment (TedLivist) [inline src/components/book/AddBook.js]: - [ ] Your inputs should be controlled and states should be the only source of truth. You can refer to the first paragraph of this [article](https://reactjs.org/docs/forms.html) to get an idea of what this means.

  As listed in the project requirements (see image below) your add book form component should have a local `state` for `title` and `author`. And that local state should be updated whenever a user updates the title and author inputs with `onchange`. This state is what should be dispatched. You can refer to this [article](https://dmitripavlutin.com/controlled-inputs-using-react-hooks/) for a quick understanding on how to create controlled components with React hooks.

  muth-req

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Emmy-github-webdev,

Good job so far!
✔️ Nice implementation of SOLID principles
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/book/AddBook.js (hunk 1 above)
Comment (Emmy-github-webdev) [reply src/components/book/AddBook.js]: Thanks, @TedLivist. recommended changes fixed.

**Review by AshakaE — APPROVED** [review]
### Hi @Emmy-github-webdev   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**